contracts/vaults/STETHVault/base/RibbonVault.sol

```diff
@@ -480,25 +479,20 @@ contract RibbonVault is
                 vaultParams.decimals
             );
 
-        // Unwrap may incur curve pool slippage
-        uint256 amountETHOut =
-            VaultLifecycleSTETH.unwrapYieldToken(
-                withdrawAmount,
-                address(collateralToken),
+        IERC20(STETH).safeTransfer(
+            msg.sender,
+            VaultLifecycleSTETH.withdrawStEth(
                 STETH,
-                STETH_ETH_CRV_POOL,
-                minETHOut
-            );
+                address(collateralToken),
+                withdrawAmount
+            )
```

Comment:
May be redundant, but to be safe, we can add a `require` that checks that the amount returned by `withdrawStEth` is equal to the withdrawAmount

```solidity
uint stEthWithdrawAmount = VaultLifecycleSTETH.withdrawStEth(
    STETH,
    address(collateralToken),
    withdrawAmount
);
require(stEthWithdrawAmount == withdrawAmount)
```

Reply:
```
// Accounts for rounding errors by a margin of 3 wei
require(_amount.add(3) >= stethBalance, "Unwrapped too much stETH");
require(_amount <= stethBalance.add(3), "Unwrapped insufficient stETH");
return stethBalance; // We return here if we have enough stETH + wstETH + ETH
```

Added this to `withdrawStEth` to account for rounding errors
Although doing the muldiv ourselves lessens to the rounding errors, there still are errors when calling `unwrap` on wstETH and `submit` on stETH
In my tests a margin of 2 wei had issues when withdrawing both stETH + wstETH and stETH + ETH, so I set it to 3 wei which worked for all conditions.